content/docs/components/pytorch.md

```diff
@@ -42,7 +42,7 @@ ks apply ${ENVIRONMENT} -c pytorch-operator
 
 ## Creating a PyTorch Job
 
-You can create PyTorch Job by defining a PyTorchJob config file. See [distributed MNIST example](https://github.com/kubeflow/pytorch-operator/blob/master/examples/tcp-dist/mnist/v1beta1/pytorch_job_mnist.yaml) config file. You may change the config file based on your requirements.
+You can create PyTorch Job by defining a PyTorchJob config file. See the manifests for the [distributed MNIST example](./examples/mnist/). You may change the config file based on your requirements.
```

Comment:
The link gives a 404 in the preview:
https://deploy-preview-715--competent-brattain-de2d6d.netlify.com/docs/components/pytorch/

I think you need to give the full URL rather than a relative one.

Reply:
Well that's embarrassing. Updated to full URL.

Reply:
LOL. Thanks!